content/docs/engine-dev/tools/tiaf/maintaining-tiaf/general-information.md

```diff
@@ -0,0 +1,114 @@
+|Safe Mode|-safemode=<on,off>|Off|Flag to specify a safe mode for tia and tiaorseed sequences where the set of unselected tests is run without instrumentation after the set of selected instrumented tests is run (this has the effect of ensuring all tests are run regardless).|
+|Draft Failing Tests|-draftfailingtests=<on,off>|Off|If enabled, will attempt to read the previous test runs data as specified in the config file and draft in any failing tests into tiaf and tiafnowrite sequences to be run in conjunction with the selected tests.|
+|Shard Tests|-shard=<on,off>|No sharding|Break any test targets with a sharding policy into the number of shards according to the maximum concurrency value.|
+|Capture Test Output|-targetout=<stdout, file>|None|Capture of individual test run stdout, where:<br /><br />`stdout` will capture each individual test target's stdout and output each one to stdout.<br /><br />`file` will capture each individual test target's stdout and output each one individually to a file.|
+|Failed Test Coverage Policy|-cpolicy=<discard, keep>|Keep|Policy for handling the coverage data of failing tests, where:<br /><br />`discard` will discard the coverage data produced by the failing tests, causing them to be drafted into future test runs.<br /><br />`keep` will keep any existing coverage data and update the coverage data for failed tests that produce coverage.|
+|Execution Failure Policy|-epolicy=<abort, continue, ignore>|Continue|Policy for handling test execution failure (test targets could not be launched due to the binary not being built, incorrect paths, etc.), where:<br /><br />`abort` will abort the entire test sequence upon the first test target execution failure and report a failure (along with the return code of the test target that failed to launch).<br /><br />`continue` will continue with the test sequence in the event of test target execution failures and treat the test targets that failed to launch as as test failures (along with the return codes of the test targets that failed to launch).<br /><br />`ignore` will continue with the test sequence in the event of test target execution failures and treat the test targets that failed to launch as as test passes (along with the return codes of the test targets that failed to launch).|
```

Comment:
update: Again reminder to escape `<` and `>` (lots of the options are missing when viewed as markdown)

Reply:
update: as as -> as

Reply:
Fixed 👍